README.md

```diff
@@ -64,6 +64,15 @@ First you'll need a `config.ini` text file with values for `github_username` and
 See [instructions on how to get a GitHub OAuth token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) (the token only needs the `public_repo` permission).
 This file is ignored in `.gitignore` for security reasons.
 
+For the MySQL database you should also add the following values:
+```
+host = 'db'
+port = '3306';
+dbname = 'nfcore';
+username = 'nfcore_admin';
+password = 'PEBBLY8exhibit_mead1cilium6despise'
```

Comment:
I tried running locally with this in my `config.ini` but am getting access denied 🤔 

```
nf-core-web       | NOTICE: PHP message: PHP Warning:  mysqli_connect(): (HY000/1045): Access denied for user 'nfcore_admin'@'172.18.0.5' (using password: YES) in /var/www/public_html/modules.php on line 3
```

Any ideas? Is your local config definitely exactly as above @mashehu?

Reply:
okay, weird syntax problem, where it looks like the environment variables have to be a mapping not an array in the composer yaml. I pushed a fix (where I also moved the database location, so it is a bit easier to debug/drop).